index.html

```diff
@@ -0,0 +1,133 @@
+                    <li><a href="/about.html">CC Campaign</a></li>
+                </ul>
+            </nav>
+            <a href="#"><img src="images/header-logo.png" alt="Global Summit Logo"></a>
+        </div>
+
+        <ul class="navbar-banner">
+            <li><a href="#"><img src="images/facebook-logo.png" class="mini-navbar-img" alt="facebook logo"></a></li>
+            <li><a href="https://twitter.com/AyokunnumiA"><img src="images/twitter-logo.png" class="mini-navbar-img"
+                        alt="twitter logo"></a></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Logo</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul>
+
+        <div>
+            <nav class="desktop-navbar">
+                <div><img src="images/header-logo.png" alt=""></div>
+                <ul class="desktop-navbar-list">
+                    <li><a href="/about.html" class="desktop-navbar-link">About</a></li>
+                    <li><a href="#" class="desktop-navbar-link">Program</a></li>
+                    <li><a href="#" class="desktop-navbar-link">Join</a></li>
+                    <li><a href="#" class="desktop-navbar-link">Sponsor</a></li>
+                    <li><a href="#" class="desktop-navbar-link">News</a></li>
+                    <li><a href="#" class="desktop-navbar-link CC-Campaign-link">CC Campaign</a></li>
+                </ul>
+            </nav>
+        </div>
+
+        <div class="desktop-header-section">
+            <div class="mobile-header-section">
+                <p class="header-theme">"You Never Fail Until You Stop Trying"</p>
+                <h1>Content Creator <br> Global Summit 2023</h1>
+                <p class="header-hero-desc">A joyful celebration believing in the value of openness and
+                    sharing, creating a positive change with people from over 80
+                    countries is taking place in October, in Korea.
+                </p>
+                <div class="date-time">
+                    <time>2023-10-15(Thurs) ~ 16(Fri)</time>
+                    <p class="venue">&#x40;National Museum of Lagos, Muson Center Lagos and more</p>
+                </div>
+            </div>
+        </div>
+    </header>
+    <main>
+        <section class="main-program-container">
+            <div>
+                <h2>Main Program</h2>
+                <span class="line"></span>
+            </div>
+            <div class="main-program-content">
+                <div class="main-program-short"><img src="images/lecture-img.png" alt="">
+                    <h3>Lecture</h3>
+                    <p class="main-program-desc">Listen to the speakers from various countries about the messages of
+                        sharing and opening</p>
+                </div>
+                <div class="main-program-short"><img src="images/exhibition-img.png" alt="">
+                    <h3>CC Exhibition</h3>
```

Comment:
- [ ] Please kindly note that for each program the white border only becomes visible when a user hovers around it as shown in the provided design template below. However what you have is the white borders displaying and no spaces between each program. Please kindly make the necessary adjustments.
- [ ] Each program should also have the appropriate background image as shown in the provided project design
- project design
![sp](https://github.com/Ayokunnumi1/Conference-Page/assets/89055502/6c096c62-514b-4573-8965-9305e404e816)
- your design
![yd](https://github.com/Ayokunnumi1/Conference-Page/assets/89055502/123c49cc-67fd-4bec-95be-2103116f3609)